photon-colors/build.js

```diff
@@ -61,52 +64,54 @@ const formats = {
     'footer': '}\n',
     'ext': 'css'
   },
-  'less': {
-    'output': [`/* Photon Colors Less Variables v${metadata.version} */\n\n`],
+  'gimp': {
+    'output': [`GIMP Palette\nName: Photon Colors\n# Photon Colors GPL Color Palette v${metadata.version}\n# ${metadata.homepage}\n\n`],
     'formatter': (color, variant, value, alpha) => {
       if (alpha == '100') {
-        return `@${color}-${variant}: ${value};\n`;
-      } else {
+        color = color.charAt(0).toUpperCase() + color.slice(1);
         const {r,g,b} = getRgb(value);
-        return `@${color}-${variant}-a${alpha}: rgba(${r}, ${g}, ${b}, ${alpha/100});\n`;
+        return `${r} ${g} ${b} ${color} ${variant}\n`;
       }
     },
-    'ext': 'less'
+    'ext': 'gpl'
   },
-  'sass': {
-    'output': [`/* Photon Colors SCSS Variables v${metadata.version} */\n\n`],
+  'ios': {
+    'output': [`/* Photon Colors JS Variables v${metadata.version}\n   From ${metadata.homepage} */\n\nextension UIColor {\n    struct Photon {\n`],
```

Comment:
The output should probably be "Photon Colors iOS Variables..." instead of "Photon Colors JS Variables..." 🙂

Reply:
Same line, is it necessary to add the homepage address here?

Reply:
Fixed, and I think it is.